docs/common/releases/release-notes.md

```diff
@@ -66,10 +66,39 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
 </dependencies>
 ```
 
-## 0.13.0 Release Notes
+## 0.13.1 Release Notes
+
+Apache Iceberg 0.13.1 was released on February 14th, 2022.
+
+**Important bug fixes:**
+
+* **Spark**
+  * [\#4023](https://github.com/apache/iceberg/pull/4023) fixes predicate pushdown in row-level operations for merge conditions in Spark 3.2. 
+  Prior to the fix, filters would not be extracted and targeted merge conditions were not pushed down leading to degraded performance
+  for these targeted merge operations.
+  * [\#4024](https://github.com/apache/iceberg/pull/4024) fixes an issue where we cannot create a table in the root namespace of the hadoop catalog.
+
+* **Flink**
+  * [\#3986](https://github.com/apache/iceberg/pull/3986) fixes an issue where there are multiple committers for multiple sink tables, 
```

Comment:
Fixes manifest location collisions when there are multiple committers in the same Flink job.

Reply:
Ah yeah, this is more concise. Thanks, and updated!